src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -208,6 +209,11 @@ export default {
         tooltipContent() {
             return this.$t('loading_external_layer')
         },
+        showSpinner() {
+            // only show the spinner for external layer, for our layers the
+            // backend should be quick enough and don't require any spinner
+            return this.layer.isLoading && this.layer.isExternal && !this.layer.hasError
```

Comment:
`isLoading` was removed, wasn't it? this might need to change to `!this.layer.kmlData` here

Reply:
No the `isLoading` is kept for external layers (WMS and WMTS), but actually for external KML it might be a good idea to kept it and to only remove it for our kmls (kml from our sys-public backend)

Reply:
you can create a getter in the KML class, something like
```
get isLoading {
    return !this.kmlData
}
```

Reply:
Sounds like a very good idea, I'll have a deeper look

Reply:
getter and setter are a very bad idea as they are not reactive in Vue which leads to subtle bug hard to find. We should use only javascript plain object as reactive data (same for store data).
See https://vuejs.org/api/options-state.html#data